docs/keps/20200619-federated-resource-status.md

```diff
@@ -189,21 +260,35 @@ The value of `status.conditions` should reflect the new state for that specific
   status:
     clusters:
     - name: "cluster3"
-      status: "ReplicaFailure"
+      status: "CreationFailed"
+      conditions:
+      - lastTransitionTime: "2020-05-25T20:23:59Z"
+        lastUpdateTime: "2020-05-25T20:23:59Z"
+        status: "True"
+        type: "NotReady"
+        reason: "ReplicaFailure"
+        message: "1 Replica is not ready, you don't have enough resources"
     conditions:
     - lastTransitionTime: "2020-05-25T20:23:59Z"
       lastUpdateTime: "2020-05-25T20:23:59Z"
-      status: "True"
-      type: "NotReady"
+      status: "False"
+      type: "Propagation"
 ```
 
-The value of `spec.conditions` contains a `NotReady` condition type, that is the
-result of checking the status of that remote `Deployment` in the target clusters.
-The value of `status.clusters.status` can be extracted from the `"ReplicaFailure"`
-status of the `Deployment` and so reused for visibility.
+The value of `status.conditions` describes the propagation error that happened to `cluster3`
+while `clusters["cluster3"].conditions` contains a `NotReady` condition type with a detailed
```

Comment:
Looking at the below type definition above, I was under the impression that this is a slice. Is that not so.
This representation is of a map. If its not a map, please update it to use an index.

```
type GenericClusterStatus struct {
	Name   string            `json:"name"`
	Status PropagationStatus `json:"status,omitempty"`
  Conditions []*metav1.Condition `json:"conditions,omitempty"`
}
```